docs/en/guide/plugins/maven.md

```diff
@@ -193,7 +193,10 @@ OS name: "windows 10", version: "10.0", arch: "amd64", family: "dos"
 ![Use of smart-doc-maven plug-in in idea](/assets/idea-maven-plugin.png)
 
 ### Plug-in source code
-[smart-doc’s maven plug-in source code](https://gitee.com/smart-doc-team/smart-doc-maven-plugin)
+
+[Gitee](https://gitee.com/TongchengOpenSource/smart-doc-maven-plugin)
+
+[GitHub](https://github.com/TongchengOpenSource/smart-doc-maven-plugin)
```

Comment:
In English documents, just provide the GitHub address. Foreign developers won't understand Gitee.